internal/pkg/action/operator_list_custom_resources.go

```diff
@@ -0,0 +1,35 @@
+package action
+
+import (
+	"context"
+
+	lib "github.com/operator-framework/kubectl-operator/pkg/action"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// OperatorListCustomResources knows how to find and list custom resources given a package name and namespace.
+type OperatorListCustomResources struct {
+	config      *Configuration
+	PackageName string
+}
+
+func NewOperatorListCustomResources(cfg *Configuration) *OperatorListCustomResources {
+	return &OperatorListCustomResources{
+		config: cfg,
+	}
+}
+
+func (o *OperatorListCustomResources) Run(ctx context.Context) (*unstructured.UnstructuredList, error) {
```

Comment:
As far as I know, this is the only function console needs. Nothing else should be exported, at least until we know there's a specific reason to make it public.